### PR TITLE
[jsk_2016_01_baxter_apc] Add node version of RBOSegmentationInBin

### PR DESCRIPTION
**Pull request by yuyu2172**
I have not decided how to update '/target_bin_name' which is a rosparam of a character in 'abcdefghijkl'.
--> WIP

### CODE REVIEW

File: jsk_2016_01_baxter_apc/node_scripts/pipeline_ros.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+from jsk_apc2016_common.msg import APCBinArray
+import rospy
+import tf2_ros
+import message_filters
+from sensor_msgs.msg import Image, CameraInfo
+from sensor_msgs.msg import PointCloud2
+from cv_bridge import CvBridge, CvBridgeError
+
+
+class PipelineROS(Pipeline):
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/node_scripts/pipeline_ros.py]: What Pipeline?

Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/node_scripts/pipeline_ros.py]: Pipeline --> SegmentationPipeline

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/node_scripts/pipeline_ros.py]: Why not InBinSegmentationPipeline?

Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/node_scripts/pipeline_ros.py]: SegmentationInBinPipeline sounds more natural to me

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/node_scripts/pipeline_ros.py]: Sounds Good.

Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/node_scripts/pipeline_ros.py]: Thanks.
I will go with PipelineSegmentationInBin

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/node_scripts/pipeline_ros.py]: Please update the title of this PR also.

**Comment by wkentaro** [comment]
This depends on other PRs, doesn't it? so [WIP]?

**Comment by yuyu2172** [comment]
The code would work no matter what order you merge the PRs.
So, this is not WIP.

File: jsk_2016_01_baxter_apc/node_scripts/pipeline_segmentation_in_bin_ros.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+from jsk_apc2016_common.spatial_feat import get_spatial_img
+from jsk_apc2016_common.mask_bin import get_mask_img
+from jsk_apc2016_common.pipeline_segmentation_in_bin\
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/node_scripts/pipeline_segmentation_in_bin_ros.py]: https://github.com/start-jsk/jsk_apc/tree/master/jsk_apc2016_common/python/jsk_apc2016_common

pipeline_segmentation_in_bin.py is missing, no?

Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/node_scripts/pipeline_segmentation_in_bin_ros.py]: I get what you are saying.
Merge after #1360.

**Comment by knorth55** [comment]
what is b/w?

**Comment by wkentaro** [comment]
between.



Sent with MailTrack
<https://mailtrack.io/install?source=signature&lang=en&referral=www.kentaro.wada@gmail.com&idSignature=22>

On Sat, Apr 30, 2016 at 2:22 PM, Shingo Kitagawa <notifications@github.com>
wrote:

> what is b/w?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/start-jsk/jsk_apc/pull/1372#issuecomment-215939464>
>



-- 
和田 健太郎 / Kentaro Wada
http://wkentaro.com

**Comment by knorth55** [comment]
+1

**Comment by knorth55** [comment]
このPRってRBOSegmentationInBinとそのRosNodeの間のInterfaceではなくRosNode自体じゃないですか？
もしそうならPRの名前がわかりにくくしている気がします

**Comment by yuyu2172** [comment]
If 'node' needs to be added in the title, then it should be 'Add node version of RBOSegmentationInBin'. 
It is the word 'interface' that we needed to remove.

**Comment by knorth55** [comment]
+1 Good

**Comment by wkentaro** [comment]
+1

**Pull request merged**